conftest.py

```diff
@@ -24,3 +26,54 @@
 logging.basicConfig(
     format='[%(asctime)s|%(name)s|%(levelname)s]: %(message)s',
     level=log_level)
+
+
+def get_task_ids_for_user(user: str):
+    """ This function uses dcos task WITHOUT the JSON options because
+    that can return the wrong user for schedulers
+    """
+    tasks = subprocess.check_output(['dcos', 'task']).decode().split('\n')
+    for task_str in tasks:
+        task = task_str.split()
+        if len(task) < 5:
+            continue
+        if task[2] == user:
+            yield task[4]
+
+
+def get_task_logs_for_id(task_id: str, lines: int=1000000):
+    return subprocess.check_output([
+        'dcos', 'task', 'log', task_id, '--lines', str(lines)]).decode()
```

Comment:
Interesting. Given that we're asking for 1000000 lines I'd expect more log output. Seeing ~ 1 MB per test failure.